scripts/sct_check_dependencies.py

```diff
@@ -17,17 +17,21 @@
 # TODO: manage .cshrc files
 # TODO: add linux distrib when checking OS
 
-
-# DEFAULT PARAMETERS
 class Param:
     # The constructor
     def __init__(self):
         self.create_log_file = 0
         self.complete_test = 0
 
 
+import commands
+import importlib
+import io
+import os
+import platform
+import sys
 
-import sys, io, os, commands, platform, importlib
+import sct_config
```

Comment:
Not really sure about the utility of `sct_config` given that most of the code inside was already in `sct_utils`.